spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require_relative '../config/environment'
```

Comment:
![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/a2ae6cd5-859d-4423-80a4-18508bcb82b7)

- [ ] It would be nice if you fix the failing tests because failing tests indicate that some of the things in your app are not working correctly 😨 Kindly fix these tests or the problems causing these tests to fail to make your app test correctly 👍